### ISSUE TITLE
Does lineage view show dependencies of datasets on dataflows?

### ISSUE THREAD

**Issue by fpk3** [MicrosoftDocs/powerbi-docs#2630]
Does lineage view show dependencies of datasets on dataflows?
If so, what are the limitations? Parameters, etc. that would prevent dependencies from being shown? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f9b34cb8-3435-8176-a8b9-2a99b2d5ce2d
* Version Independent ID: 3d7b6b44-02c2-1a66-0672-794dd1c3975b
* Content: [Data lineage - Power BI](https://docs.microsoft.com/en-us/power-bi/collaborate-share/service-data-lineage)
* Content Source: [powerbi-docs/collaborate-share/service-data-lineage.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/master/powerbi-docs/collaborate-share/service-data-lineage.md)
* Service: **powerbi**
* Sub-service: **powerbi-eim**
* GitHub Login: @paulinbar
* Microsoft Alias: **painbar**

**Comment by paulinbar** [comment]
Hi @fpk3, thanks for reaching out to us with your question. I am going to forward it on to the product manager and will get back to you. Probably won't be before Sunday though.

**Comment by mihart** [comment]
Hi @fpk3 ,
I reached out to the Product team and here is their response.
The limitations are documented on our Dataflows limitation doc (see link below). Currently the lineage view scope for dataflows is limited to what exists in the current workspace.

https://docs.microsoft.com/en-us/power-bi/transform-model/dataflows/dataflows-features-limitations

I've ensured that there is a link from this doc to the Limitations doc.  Thank you for your suggestions.
All the best,
the DOCS team
